.travis.yml

```diff
@@ -1,8 +1,14 @@
 language: node_js
 node_js:
-- '0.10'
+- '0.12'
+before_install:
+- phantomjs -v
 notifications:
   email: false
+sudo: false
```

Comment:
This is awesome news. It says that we are guaranteed <1 min queue time? Is that correct? If it is, that's amazing.